README.md

```diff
@@ -131,7 +131,7 @@ setTimeout(restart, 86400000);
 **Good:**
 
 ```ts
-// Declare them as capitalized named constants.
+// Declare them as capitalized symbolic constants.
```

Comment:
How is _symbolic constants_ better than _name constants_? The use of term _symbolic_ might confuse people to use `Symbol`, which is not the intention.

Reply:
The term _symbolic constants_ is widely used throughout SE. However, seeing your next comment I do understand that the intention is to have a more straight-forward way for beginners to better get up to speed.

Is there any way we can incorporate both ? Something like the below for instance:

```
// Declare them as capitalized symbolic constants (also known as symbolic constants).
```

Reply:
The term _symbolic constant_ is more  oftenly used in C/C++, you would rarely see it in a JS/TS reference. It is also mentioned as _literal constant_ in other places. I prefer to keep it simple and not confuse the reader with too much terminology.

Reply:
Addressed on 36d14bf048c083c2a01b3c448f9a3c9bab80757e